Makefile

```diff
@@ -216,13 +216,14 @@ verify: .init verify-generated verify-client-gen verify-docs verify-vendor
 	@rm .out
 	@#
 	@echo Running golint and go vet:
-	@# Exclude the generated (zz) files for now, as well as defaults.go (it
+	@# Exclude the generated (zz) files in pkg and svcat for now, as well as defaults.go (it
 	@# observes conventions from upstream that will not pass lint checks).
 	@$(DOCKER_CMD) sh -c \
 	  'for i in $$(find $(TOP_SRC_DIRS) -name *.go \
 	    | grep -v ^pkg/kubernetes/ \
 	    | grep -v generated \
 	    | grep -v ^pkg/client/ \
+	    | grep -v pkg/svcat/service-catalog/fakes \
```

Comment:
How hard is it to regenerate the fake? This shouldn't be required as of https://github.com/maxbrunsfeld/counterfeiter/pull/66 http://gopkg.in/maxbrunsfeld/counterfeiter.v5

Reply:
you just run `counterfeiter . SvcatAPI` in the pkg/svcat/service-catalog directory. That line is needed otherwise the linter complains about the lack of comments in the auto generated code.

Reply:
I get super different output. Can you try with http://gopkg.in/maxbrunsfeld/counterfeiter.v5 ? It should remove the need to add this extra ignore.

Reply:
Yeah I didn't realize my counterfeiter was out of date. Updated it and regenerated the fake.

Reply:
take the ignore out, and it should work.